From: Andreas Herrmann <andreas.herrmann3@amd.com>
Subject: and
References: fate#311968
Patch-mainline: v2.6.37-rc1

Signed-off-by: Thomas Renninger <trenn@suse.de>

The file names are somehow misleading as the code is not specific to
AMD K8 CPUs anymore. The files accomodate code for other AMD CPU
northbridges as well.

Same is true for the config option which is valid for AMD CPU
northbridges in general and not specific to K8.

Signed-off-by: Andreas Herrmann <andreas.herrmann3@amd.com>
LKML-Reference: <20100917160343.GD4958@loge.amd.com>
Signed-off-by: H. Peter Anvin <hpa@linux.intel.com>
Automatically created from "patches.arch/0018-x86-k8-Rename-k8.-ch-to-amd_nb.-ch-and.patch" by xen-port-patches.py

--- sle11sp2-2011-07-15.orig/arch/x86/kernel/setup-xen.c	2011-07-15 14:22:08.000000000 +0200
+++ sle11sp2-2011-07-15/arch/x86/kernel/setup-xen.c	2011-07-15 14:22:13.000000000 +0200
@@ -106,7 +106,7 @@
 #include <asm/percpu.h>
 #include <asm/topology.h>
 #include <asm/apicdef.h>
-#include <asm/k8.h>
+#include <asm/amd_nb.h>
 #ifdef CONFIG_X86_64
 #include <asm/numa_64.h>
 #endif
